doc/man3/s2i_ASN1_IA5STRING.pod

```diff
@@ -29,14 +31,19 @@ i2s_ASN1_ENUMERATED_TABLE,
  char *i2s_ASN1_ENUMERATED_TABLE(X509V3_EXT_METHOD *method,
                                 const ASN1_ENUMERATED *e);
 
+ char *i2s_ASN1_UTF8STRING(X509V3_EXT_METHOD *method,
```

Comment:
As you're adding those functions, please add HISTORY section.

Reply:
I've added the HISTORY section saying:
`i2s_ASN1_UTF8STRING() and s2i_ASN1_UTF8STRING() were made public in OpenSSL 3.0.`